particles/Tutorial/Javascript/Demo/Game.arcs

```diff
@@ -0,0 +1,60 @@
+// Tutorial Demo
+// Shows how to use Arcs features to create a tic-tac-toe game.
+import 'TicTacToe.arcs'
+import 'GameSchemas.arcs'
+
+particle Game in './source/Game.js'
+  inout Person humanPlayer
+  inout Person computerPlayer
+  in GameState {gameOver} gameState
+  in Move {Text move} humanMove
```

Comment:
`Text` is inferred from the `Move` schema that's in the context, you can just do `Move {move}`

Let's do inline schemas for the rest of connections as well. I will land the warning for non-inline schemas soon and Myk is doing type slicing so that you only get what you declare - we're all invested in this! :D